crypto/nrf_cc312_platform/include/nrf_cc3xx_platform_kmu.h

```diff
@@ -48,25 +82,66 @@
      *       the AES key is split between two slots in KMU
      */
     #define NRF_CC3XX_PLATFORM_KMU_AES_ADDR_2               (0x50845410UL)
+
+    /** @brief Address of the ChaChaPoly key register in CryptoCell for 128 bit keys */
+    #define NRF_CC3XX_PLATFORM_KMU_CHACHAPOLY_ADDR          (0x50845388UL)
+
+    /** Address of the first 128 bits of ChaChaPoly key in CryptoCell */
+    #define NRF_CC3XX_PLATFORM_KMU_CHACHAPOLY_ADDR1         (0x50845388UL)
+
+    /** Address of the last 128 bits bits of ChaChaPoly key register in CryptoCell HW
+     *
+     * @note Used only when ChaChaPoly key is larger than 128 bits, in which case
+     *       the ChaChaPoly key is split between two slots in KMU
+     */
+    #define NRF_CC3XX_PLATFORM_KMU_CHACHAPOLY_ADDR2         (0x50845398UL)
 #endif
 
+/** @brief Write a key into the KMU
+ *
+ * @details This writes a key into the KMU slot defined by slot_id. If the key is larger than
+ *          128 bit it will occupy two KMU consecutive KMU slots.
+ *
+ * @note    This function use the default kmu key permissions,
```

Comment:
```suggestion
 * @note    This function use the default KMU key permissions,
```

Reply:
Will be fixed, also will be added to 0.9.10 release as well

Reply:
Done